app/uk/gov/hmrc/cataloguefrontend/shuttering/ShutterStateModel.scala

```diff
@@ -133,6 +133,17 @@ case class ShutterState(
         case Environment.Staging      => staging
         case Environment.Dev          => development
       }
+
+    def isShuttered(env: String): Option[Boolean] =
+      env match {
+        case "Production"    => Some(production.isInstanceOf[ShutterStatus.Shuttered])
+        case "External Test" => Some(externalTest.isInstanceOf[ShutterStatus.Shuttered])
+        case "QA"            => Some(qa.isInstanceOf[ShutterStatus.Shuttered])
+        case "Staging"       => Some(staging.isInstanceOf[ShutterStatus.Shuttered])
+        case "Development"   => Some(development.isInstanceOf[ShutterStatus.Shuttered])
+        case _               => None
+      }
+
```

Comment:
Could this be simplified to remove the isInstanceOf's? 

e.g.: 
`def isShuttered(env: String) = Environment.parse(env).exists(e => statusFor(e).value.asString == ShutterStatusValue.Shuttered.asString)`

or something

Reply:
Seems to work.